packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -54,45 +54,19 @@
 
         <v-tab name="Trace View" :is-active="isViewingFlow" :ref="VIEW_FLOW">
           <div class="trace-view">
-            <div class="trace-view__search">
-              <div class="trace-view__search-input-wrap">
-                <span class="trace-view__search-input-prefix">
-                  <SearchIcon />
-                </span>
-                <input
-                  v-model="traceFilterValue"
-                  class="trace-view__search-input-element"
-                  type="text"
-                  autocomplete="off"
-                  placeholder="search events..."
-                />
-                <span
-                  v-if="traceFilterValue"
-                  class="trace-view__search-input-suffix"
-                  @click="clearTraceFilterInput"
-                >
-                  <CloseThinIcon />
-                </span>
-              </div>
-              <div
-                class="trace-view__search-arrows"
-                v-if="highlightedNodes.length"
-              >
-                <div class="trace-view__search-arrow" @click="prevTraceFilter">
-                  <ArrowSearchLeftIcon />
-                </div>
-                <div class="trace-view__search-arrows-text">
-                  <span v-if="Number.isInteger(currentTraceFilterIndex)">
-                    <b>{{ currentTraceFilterIndex + 1 }}</b>
-                    /
-                  </span>
-                  {{ highlightedNodes.length }} results
-                </div>
-                <div class="trace-view__search-arrow" @click="nextTraceFilter">
-                  <ArrowSearchRightIcon />
-                </div>
-              </div>
-            </div>
+            <v-trace-filter
+              ref="traceFilter"
+              :nodesLength="highlightedNodes.length"
+              :currentIndex="currentTraceFilterIndex"
+              :suggestions="eventsSuggestions"
+              @onChange="
+                (value) => {
+                  this.traceFilterValue = value;
```

Comment:
Seems unrelated to this change, but there are some casing issues with the terms being searched for.

Searching for `SELECT` results in nothing. It should have many results.
Searching for `select` seems to yield the intended results, despite the actual text being `SELECT`.

https://user-images.githubusercontent.com/8737782/152196173-e30ce147-e2b5-42db-9f78-7ee392a93164.mp4

Reply:
Fixed, full-text search is case-insensitive now https://github.com/applandinc/appmap-js/pull/534/commits/b591f836495be962b38cc25908767581ca2926d1